examples/login.ts

```diff
@@ -21,14 +22,21 @@ async function main() {
   const objective = "please login into the website";
   const maxIterations = 10;
 
+  const providerOptions = {
+    apiKey: process.env.ANTHROPIC_API_KEY!,
+    provider: "anthropic",
+  };
   const logger = new Logger("info");
-  const openAIChatApi = new OpenAIChatApi(
-    {
-      apiKey: process.env.OPENAI_API_KEY,
-    },
-    { model: "gpt-4" }
-  );
-  const agent = new Agent(openAIChatApi);
+  const chatApi = completionApiBuilder(providerOptions, {
+    model: "claude-2.1",
```

Comment:
but claude 3 came out today! heh

Reply:
Oh is that what you meant by "the new Anthropic models"?

Reply:
the `llm-api` breaks when trying any of the claude-3 models :((